client/packages/common/src/hooks/useNativeClient/useNativeClient.ts

```diff
@@ -112,15 +115,15 @@ export const useNativeClient = ({
 
   const allowSleep = async () => {
     // Currently only supported on native platforms via capacitor
-    if (!Capacitor.isNativePlatform) return;
+    if (!Capacitor.isNativePlatform()) return;
```

Comment:
Was this a bug before? or did this change through the plugin?

Reply:
An existing bug that I'd spotted when editing the file